docs/source/index.rst

```diff
@@ -0,0 +1,47 @@
+.. Neffint documentation master file, created by
+   sphinx-quickstart on Thu Jun 22 18:09:09 2023.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to Neffint's documentation!
+===================================
+
+Neffint is an acronym for **N**\ on-\ **e**\ quidistant **F**\ ilon **F**\ ourier **int**\ egration. This is a python package for computing Fourier integrals using a method based on Filon's rule with non-equidistant grid spacing.
+
+Neffint licensed under an Apache-2.0 license. See `HERE <https://www.apache.org/licenses/LICENSE-2.0>`_ for details.
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Contents:
+
+   theory
+   ex_fixed_grid_FI.ipynb
+   ex_adaptive.ipynb
+   api
+
+
+Installation procedure
```

Comment:
we could also provide the installation procedure for 1) compiling the doc, and 2) launching the tests

Reply:
Will do